src/alerting/reducers/alertBuilder.test.ts

```diff
@@ -27,14 +31,16 @@ import {
   DeadmanCheck,
 } from 'src/types'
 
+const noopAction: Action = {type: undefined}
+
 const check_1 = {
-  ...(CHECK_FIXTURE_1 as ThresholdCheck),
+  ...((CHECK_FIXTURE_1 as any) as ThresholdCheck),
```

Comment:
is the double casting here required? I'm not sure why we would need to cast `as any`

Reply:
without this casting TS compiler throws error 
```
Conversion of type 'import("/src/client/generatedRoutes").ThresholdCheck' to type 'import("/src/types/alerting").ThresholdCheck' may be a mistake because neither type sufficiently overlaps with the other. If this was intentional, convert the expression to 'unknown' first.
  Property 'activeStatus' is missing in type 'ThresholdCheck' but required in type 'CheckOverrides'.
```